### PR TITLE
Fix broken pane toggle

### PR DESCRIPTION
**Pull request by roemhildtg**
Panes are not toggleable by the button since https://github.com/cmv/cmv-app/pull/607

### CODE REVIEW

File: viewer/js/viewer/_LayoutMixin.js [hunk 1]
```diff
@@ -232,11 +232,7 @@ define([
                     // Toggle
                     newDisp = (oldDisp === 'none') ? 'block' : 'none';
                 } else {
-                    this.handleError({
-                        source: '_LayoutMixin',
-                        error: 'Invalid type passed as "show" property of "togglePane" function : ' + typeof(show)
-                    });
-                    return;
+                    newDisp = (domStyle.get(domNode, 'display') === 'none') ? 'block' : 'none';
```
Comment (amcgee) [inline viewer/js/viewer/_LayoutMixin.js]: This is logically identical to the above case, so if we want to accept arbitrary input for the 'see' parameter we should just remove this error condition and change the `else if (show === undefined)` case to `else`

**Comment by roemhildtg** [comment]
Agreed, I like the second approach:
```
on(this.collapseButtons[key], 'click', lang.hitch(this, 'togglePane', key, null, false));
```

**Comment by tmcgee** [comment]
I like the second approach as well. 

For consistency, the `suppressEvents` parameter should also be passed to `togglePane` method in the [response to the topic](https://github.com/cmv/cmv-app/pull/613/files#diff-2af1b0fe91ac5fcca03fad2cd47d9b11R78) `viewer/togglePane`.

**Comment by amcgee** [comment]
Agreed!

**Comment by roemhildtg** [comment]
Just noticed, you guys both have really similar usernames...are you related? 😃

**Comment by tmcgee** [comment]
@roemhildtg can you also add the additional argument [here](https://github.com/cmv/cmv-app/blob/develop/viewer/js/viewer/_LayoutMixin.js#L78).

**Comment by DavidSpriggs** [comment]
@amcgee merge base branch. thanks guys!

**Comment by tmcgee** [comment]
^^^ @roemhildtg that comment is for you.

**Comment by DavidSpriggs** [comment]
yep! sorry @roemhildtg my bad, thanks!

**Comment by roemhildtg** [comment]
Sorry! So just to be sure, that should be changed to `this.togglePane(args.pane, args.show, false);` correct?

**Comment by tmcgee** [comment]
@roemhildtg how about passing `args.suppressEvents`?

**Comment by roemhildtg** [comment]
Okay, gotcha.

**Pull request merged**